src/bun.js/http.exports.js

```diff
@@ -292,6 +486,209 @@ export class IncomingMessage extends Readable {
   }
 }
 
+function emitErrorNt(msg, err, callback) {
+  callback(err);
+  if (typeof msg.emit === "function" && !msg._closed) {
+    msg.emit("error", err);
+  }
+}
+
+function onError(self, err, cb) {
+  process.nextTick(() => emitErrorNt(self, err, cb));
+}
+
+function isUint8Array(arr) {
+  return Object.prototype.toString.call(arr) === "[object Uint8Array]";
+}
+
+function write_(msg, chunk, encoding, callback, fromEnd) {
+  if (typeof callback !== "function") callback = nop;
+
+  let len;
+  if (chunk === null) {
+    // throw new ERR_STREAM_NULL_VALUES();
+    throw new Error("ERR_STREAM_NULL_VALUES");
+  } else if (typeof chunk === "string") {
+    len = Buffer.byteLength(chunk, encoding);
+  } else if (isUint8Array(chunk)) {
```

Comment:
Buffer.byteLength on a Uint8Array returns the byte length of the Uint8Array, meaning this check is unnecessary